Jenkinsfile

```
@@ -935,7 +935,7 @@ pipeline {
                                            export PDSH_SSH_ARGS_APPEND="$SSH_KEY_ARGS"
                                            test_tag=$(git show -s --format=%B | sed -ne "/^Test-tag:/s/^.*: *//p")
                                            if [ -z "$test_tag" ]; then
-                                               test_tag=regression,vm
+                                               test_tag=pr
```

Ahhh.  I think I misled you here.  Since this is the VM-test stage, we need to exclude tests that require hardware with `-hw`.

```suggestion
                                               test_tag=pr,-hw
```